compass/ocean/tests/ice_shelf_2d/__init__.py

```diff
@@ -14,11 +14,14 @@ def __init__(self, mpas_core):
         """
         super().__init__(mpas_core=mpas_core, name='ice_shelf_2d')
 
-        for resolution in ['5km']:
-            for coord_type in ['z-star', 'z-level']:
+        for resolution in [2e3, 5e3]:
+            for coord_type in ['z-star', 'z-level', 'single_layer']:
```

Comment:
Have we used this weird mix of `-` and `_` elsewhere?  I guess I would be slightly inclined toward just `-` but it's not a big deal.

Reply:
I think I originally used `single_layer` with `drying_slope`. I thought we wanted to use underscores most places and were maybe only using hyphens for `z-star` and `z-level` because they are a single word?

Reply:
Makes sense, let's leave it.